### ISSUE TITLE
Merge intro pages of new API explorers with existing ones and clean them up

### ISSUE THREAD

**Issue by christian-konrad** [camunda/camunda-docs#3585]
C.f. https://github.com/camunda/camunda-docs/pull/3569#issuecomment-2036337419

![image](https://github.com/camunda/camunda-docs/assets/56470362/31314a28-0a96-4753-b7f5-6e26cf172da5)

![image](https://github.com/camunda/camunda-docs/assets/56470362/cb9933c4-46de-4f68-8a81-e260358a23f6)

Each REST API now has two intro pages: The hand-written overview page, and the intro page auto-generated from OpenAPI specs.

We should consider merging them, or probably just not generate the auto-generated one. It adds one extra page for the user to navigate without any new info (the auth methods are also documented on dedicated auth pages).

**Comment by christian-konrad** [comment]
@pepopowitz isn't there a way to hook into the generator to skip that page, or hook into a step afterwards to delete it again?

**Comment by pepopowitz** [comment]
> @pepopowitz isn't there a way to hook into the generator to skip that page, or hook into a step afterwards to delete it again?

I don't think I want to skip or delete that page. It does contain valuable information, re: the authentication schemes that are available, and license. 

What do you think about renaming the page from Introduction to something else?

**Comment by akeller** [comment]
Looks like this was completed. @pepopowitz can you confirm?

**Comment by pepopowitz** [comment]
No, I don't know of any work that was completed (or even started) here.

**Comment by christinaausley** [comment]
The Tasklist API (REST) is now deprecated. Closing.

**Comment by christinaausley** [comment]
Work on this has already been completed/related APIs are deprecated.
